default.nix

```diff
@@ -1,4 +1,13 @@
-{ callPackage, python3, python2, lua5_1, readline, clojure, tcl }:
+{ pkgs ? import <nixpkgs>{} } :
+let
+    callPackage = pkgs.callPackage;
```

Comment:
Nit: instead of defining these here, we could use `pkgs` inline, ie:
```nix
prybar-python2 = buildPrybar { language = "python2"; buildInputs = [ pkgs.python2 ]; };
```